docs/installation/docker-compose.md

```diff
@@ -138,57 +132,140 @@ docker-compose up -d
+### Rebuild docker-compose.yml
 
-## Updating ISLE
+`make -B docker-compose.yml` will rebuild your `docker-compose.yml` file based on configuration in your `.env` file.  `make demo` and `make local` will automatically do this for you, but if you change configuration at a later point in time, you'll need to run this command to see your changes reflected.
+
+### Config Export
+
+`make config-export` will export your site's configuration to your site's config sync directory (usually `config/sync` inside your Drupal root folder).
+
+### Config Import
+
+`make config-import` will import site's configuration from your site's config sync directory (usually `config/sync` inside your Drupal root folder).
+
+### Database Dump
+
+`make drupal-database-dump DEST=/your/path/dump.sql` will dump your Drupal database and place the file at `DEST`.
+
+### Database Import
+
+`make drupal-database-import SRC=/your/path/dump.sql` will import your Drupal database from the file at `SRC`.
+
+### Export Public Files
+
+`make drupal-public-files-dump DEST=/your/path/public_files.tgz` will export your public filesystem and place it as a single zipped tarball at `DEST`. 
+
+### Import Public Files
+
+`make drupal-public-files-import SRC=/your/path/public_files.tgz` will import your public filesystem from a single zipped tarball at `SRC`. 
+
+### Fcrepo Export
 
-Documentation to come - see [Issue #121](https://github.com/Islandora-Devops/isle-dc/issues/121)
+`make fcrepo-export DEST=/your/path/fcrepo-export.tgz` will export your Fedora repository and place it as a single zipped tarball at `DEST`
 
-### Drupal Updates
+### Fcrepo Import
 
-to come
+`make fcrepo-import SRC=/your/path/fcrepo-export.tgz` will import your Fedora repository from a single zipped tarball at `SRC`
 
-### Module Updates (non-Islandora)
+### Reindex Fedora Metadata
+  
+`make reindex-fcrepo-metadata` will reindex RDF metadata from Drupal into Fedora.
 
-to come
+### Reindex Solr
+`make reindex-solr` will rebuild rebuild Solr search index for your repository. 
 
-### Module Updates (Islandora)
+### Reindex the Triplestore
+
+`make reindex-triplestore` will reindex RDF metadata from Drupal into Blazegraph. 
+
+## Developing with ISLE 
+
+If you used `make local` then the drupal root folder is in a new directory in the isle-dc folder named `codebase`. This is live and editable in whatever development environment you would like.
+
+### Embedded Editor
+
+If you 
+
+- Used `make demo`
+- Are using a custom Drupal image
+- Don't want to bother with setting up a development environment right now
+
+You can use an embedded VSCode that's available at port 8443. To enable it
+
+- Set `INCLUDE_CODE_SERVER_SERVICE=true` in your `.env` file
+- make -B docker-compose.yml
+- docker-compose up -d  
+
```

Comment:
Is it possible to use code server with a `make local` install and codebase directory?